packages/manager-react-components/src/components/navigation/menus/action/action.component.tsx

```diff
@@ -23,11 +23,15 @@ export interface ActionMenuItem {
   className?: string;
 }
 
+export enum ActionMenuMode {
+  STANDARD = 'standard',
+  DATAGRID = 'datagrid',
+  CARD = 'card',
```

Comment:
This component need not know the parent components which are using it. So can we make these mode more generic? 
For example, this configuration will not be scalable if we want to use the same display as Datagrid in other component like form or so on.

Reply:
Using the property 'variant' is not enough ? Or we want to be more restrictive ?

Reply:
I think this component has 3 use cases and the current modes make it obvious what mode you need to choose for which use case, but standard, which I didn't find a better name

please provide me mode names if you have an idea

Reply:
would "STANDARD", "COMPACT_OUTLINE" and "COMPACT_GHOST" be better ?